backend/controller/cronjobs/cronjobs.go

```diff
@@ -115,8 +120,16 @@ func (s *Service) scheduleCronJobs(ctx context.Context) (err error) {
 	}
 	logger.Tracef("Scheduling %d cron jobs", len(jobs))
 	for _, job := range jobs {
-		err = s.scheduleCronJob(ctx, tx, job)
+		err = s.scheduleCronJob(ctx, tx, job, now)
 		if err != nil {
+			s.timelineService.EnqueueEvent(ctx, &timeline.CronScheduled{
+				DeploymentKey: job.DeploymentKey,
+				Verb:          job.Verb,
+				StartTime:     now,
+				ScheduledAt:   job.NextExecution,
+				Schedule:      job.Schedule,
+				Error:         optional.Some(err.Error()),
+			})
```

Comment:
Would it make sense to have these `EnqueueEvent` calls inside of the `scheduleCronJob` func?

Reply:
Yeah guess I could. It'll just be 8 places that call `EnqueueEvent` instead of 2

Reply:
ah ok! this is cool if it's best 👍 I wasn't quite sure on that one :)